WordPress/UITests/Tests/PostTests.swift

```diff
@@ -0,0 +1,39 @@
+import UITestsFoundation
+import XCTest
+
+class PostTests: XCTestCase {
+    override func setUpWithError() throws {
+        try super.setUpWithError()
+        setUpTestSuite()
+
+        try LoginFlow.login(
+            siteUrl: WPUITestCredentials.testWPcomSiteAddress,
+            email: WPUITestCredentials.testWPcomUserEmail,
+            password: WPUITestCredentials.testWPcomPassword,
+            title: WPUITestCredentials.testWPcomSiteForScheduledPost
+        )
+
+        try TabNavComponent()
+            .goToBlockEditorScreen()
+    }
+
+    override func tearDownWithError() throws {
+        try super.tearDownWithError()
+        takeScreenshotOfFailedTest()
+    }
+
+    let title = "Scheduled Post"
```

Comment:
Similar thought here: changing to `postTitle` would be more descriptive 🤔

Reply:
in `EditorGutenbergTests`, we're also using `title` for the post title, which for me was clear enough since we know what the test class is for. 

but i can update it to be more descriptive, and i'll update both for consistency.